### ISSUE TITLE
Error if account has no email

### ISSUE THREAD

**Issue by acsfer** [rullzer/twofactor_email#31]
If the user has no email in his account, `GET /settings/user/security` trows this error:

```
[index] Error: Exception: Argument 1 passed to OCA\TwoFactorEmail\EmailMask::maskEmail() must be of the type string, null given, called in /var/www/nextcloud/apps/twofactor_email/lib/Provider/State.php on line 80 at <<closure>>

0. /var/www/nextcloud/lib/private/AppFramework/App.php line 126
   OC\AppFramework\Http\Dispatcher->dispatch(OC\Settings\Cont ... {}, "index")
1. /var/www/nextcloud/lib/private/AppFramework/Routing/RouteActionHandler.php line 47
   OC\AppFramework\App::main("OC\\Settings\\C ... r", "index", OC\AppFramework\ ... {}, {section: "secur ... "})
2. <<closure>>
   OC\AppFramework\Routing\RouteActionHandler->__invoke({section: "secur ... "})
3. /var/www/nextcloud/lib/private/Route/Router.php line 297
   undefinedundefinedcall_user_func(OC\AppFramework\ ... {}, {section: "secur ... "})
4. /var/www/nextcloud/lib/base.php line 1000
   OC\Route\Router->match("/settings/user/security")
5. /var/www/nextcloud/index.php line 42
   OC::handleRequest()
```

**Comment by J0WI** [comment]
I just run into the same issue.

**Comment by dabosch** [comment]
Same here, on Nextcloud 18.0.3

**Comment by florianwgnr** [comment]
Still a problem, even the "Security" page is not working when a user has no email.

**Comment by RoelReijn** [comment]
Same problem over here. Running Nextcloud 18.0.6.

**Comment by angerized** [comment]
> 
> 
> Same problem over here. Running Nextcloud 18.0.6.

Hello, I can confirm @RoelReijn edit fixes the issue, and that it works with 19.0.5. Though I had to edit the file manually.

Can maybe @rullzer update the extension to make it compatible with <NC 19 and including this fix?

**Comment by J0WI** [comment]
Fixed in df7d3ad3f207ab8a41c9a7a5e0f1c3ae62b4ec50
